README.md

```diff
@@ -1,46 +1,46 @@
-# Project-Webpack
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-A basic webpack boilerplate, ready to be used immediately, no configuration needed. It doesn't tell you, which framework or library you should be using, it contains only a handful of modules:
+# Cinemash App(JS Capstone Project)
 
-- Webpack
-- html-webpack-plugin
-- css-loader
-- style-loader
-- webpack-dev-server
+![Cinemash](https://user-images.githubusercontent.com/71140133/153568771-ab652ae7-d6c1-4e68-b818-9be10af0414b.PNG)
+
+
+The cinemash app is an online curation of blockbuster movies. It allows users to view a library of their favorite movies.
+Cinemash users are able to comment, like and make reservations for their favorite movies.
+
+## Presentation Video
 
-Everything else is up to you.
+[Link to presentation video](https://drive.google.com/file/d/1Os5CsONCsoRRJK_VaGkaeFcQMiIkJW20/view?usp=sharing)  
 
-## Usage
+## Live Demo
 
-start by cloning this repository by running:
+[Demo](https://cinemash.netlify.app/)
 
-```
-git clone https://github.com/Yerimah/Project-Webpack.git
-```
+## Built With
 
-cd to the folder
+- HTML 
+- CSS
+- Javascript
+- Webpack
+- Kanban board
+
+## Getting Started
 
-```
-cd Project-Webpack
-```
+To get a local copy up and running follow these simple example steps.
 
-run this command to install all the packages needed
+### Prerequisites
 
-```
-npm install
-```
+clone repo: `git clone https://github.com/Yerimah/JS-Capstone-Project`
 
-to run a live server on port 8080 run this command
+then
+`cd JS-Capstone-Project`
 
-```
-npm run start
-```
+To run locally
+`npm run start`
 
-to build your project run:
+### Install
 
-```
-npm run build
-```
+run `npm install` to install dependencies
```

Comment:
- [ ] Your users will first need to install the dependencies, and then run `npm start` so the project works as intended on their end.  Kindly correct this, so your users can follow the instructions to the letter without room for confusion 👍

Reply:
Resolved ✔️